pkgs/development/python-modules/dedupe-pyhacrf/default.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  cython,
+  numpy,
+  setuptools,
+  wheel,
+
+  # dependencies
+  dedupe-pylbfgs,
+
+  # tests
+  pytestCheckHook,
+}:
+
+buildPythonPackage {
+  pname = "dedupe-pyhacrf";
+  version = "0.2.8-unstable-2025-05-16";
+  pyproject = true;
+
+  # NOTE: This is a fork of dirko/pyhacrf maintained by dedupeio
+  # PyPI package name: pyhacrf-datamade
```

Comment:
So pname and attribute name should be the same: https://nixos.org/manual/nixpkgs/stable/#contributing